specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-07-01-preview/dataConnectorDefinitions.json

```diff
@@ -0,0 +1,758 @@
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/dataConnectorDefinitions": {
+      "get": {
+        "x-ms-examples": {
+          "Get all data connector definitions.": {
+            "$ref": "./examples/dataConnectorDefinitions/GetDataConnectorDefinitions.json"
+          }
+        },
+        "tags": [
+          "ConnectorDefinitions"
+        ],
+        "description": "Gets all data connector definitions.",
+        "operationId": "DataConnectorDefinitions_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ConnectorDefinitionBaseArmCollectionWrapper"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/dataConnectorDefinitions/{dataConnectorDefinitionName}": {
+      "get": {
+        "x-ms-examples": {
+          "Get customize data connector definition": {
+            "$ref": "./examples/dataConnectorDefinitions/GetCustomizableDataConnectoeDefinitionById.json"
+          }
+        },
+        "tags": [
+          "ConnectorDefinitions"
+        ],
+        "description": "Gets a data connector definition.",
+        "operationId": "DataConnectorDefinitions_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "in": "path",
+            "name": "dataConnectorDefinitionName",
```

Comment:
Consider add it to the common definitions, because it's used multiple times.

Reply:
Fixed